rn/Teacher/src/modules/address-book/AddressBook.js

```diff
@@ -197,6 +197,7 @@ export class AddressBook extends Component<AddressBookProps, State> {
     return (
       <Screen
         drawUnderNavBar
+        navBarStyle='global'
```

Comment:
I think the style we want for these is `'modal'`. The bar background will end up white, with the buttons tinted the linkColor.

Reply:
Modal just displayed the standard iOS style. Is this a bug? 
![Simulator Screen Shot - iPhone 11 - 2020-10-19 at 15 59 04](https://user-images.githubusercontent.com/72396990/96461146-5048c100-1224-11eb-853d-246c3c888326.png)

Reply:
That looks correct, assuming the brand for your sandbox has electric blue for the linkColor.

Reply:
Yes, that was the setting, thanks. I forgot these are used in the app as well. I've modified the PR accordingly.